core/auth/src/test/java/software/amazon/awssdk/auth/credentials/AwsCredentialsProviderChainTest.java

```diff
@@ -114,13 +114,57 @@ public void testGetCredentialsException() {
                                                                        .credentialsProviders(provider1, provider2)
                                                                        .build();
 
-        thrown.expect(SdkClientException.class);
-        thrown.expectMessage(provider1.exceptionMessage);
-        thrown.expectMessage(provider2.exceptionMessage);
+        SdkClientException e = assertThrows(SdkClientException.class, () -> chain.resolveCredentials());
+        assertThat(e.getMessage()).contains(provider1.exceptionMessage);
+        assertThat(e.getMessage()).contains(provider2.exceptionMessage);
+    }
 
-        chain.resolveCredentials();
+    @Test
+    public void testEmptyChain() {
```

Comment:
Optional Suggestion
can we make it clear in the test API name like
`emptyProviderChain_ThorwsException`
I feel the test prefix to test case API name is redundant.

Reply:
I kept the convention of the file so far, to avoid making the change bigger. I'll go ahead and follow the [convention](https://github.com/aws/aws-sdk-java-v2/blob/master/docs/design/NamingConventions.md) across the file.